src/en/administration/install_docker.md

```diff
@@ -21,6 +21,11 @@ Open up your `docker-compose.yml`, and make sure `LEMMY_EXTERNAL_HOST` for `lemm
 ```
 - LEMMY_INTERNAL_HOST=lemmy:8536
 - LEMMY_EXTERNAL_HOST=your-domain.com
+```
+
+You may also want to disable HTTPS for local development:
+
+```
```

Comment:
This setup is not meant for development, we have separate documentation for that. So i think you should just remove that line.

https://join-lemmy.org/docs/en/contributing/docker_development.html